### ISSUE TITLE
thresholds updated in package.json when --configPath is provided

### ISSUE THREAD

**Issue by provthomas** [Koleok/jest-coverage-ratchet#8]

**Comment by ayansome1** [comment]
I am facing the same issue

**Comment by JonathanATaylor** [comment]
Yep, I'm experiencing this as well. Judging by line 33 of src/program.js, the --configPath is only used to read the original thresholds. When writing the new thresholds, the app goes straight for the package.json:
` R.chain(thresholds => F.both(getPackageJsonPath(), F.of(thresholds))),`
Surely this not the desired behavior, as it results in having coverage thresholds set in two different files.
